app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/projectActionButton.tsx

```diff
@@ -23,136 +22,147 @@ enum SpinnerState {
 // =================
 
 /** The interval between requests checking whether the IDE is ready. */
-const STATUS_CHECK_INTERVAL = 10000
+const CHECK_STATUS_INTERVAL = 10000
+/** The interval between requests checking whether the VM is ready. */
+const CHECK_RESOURCES_INTERVAL = 5000
```

Comment:
1. These values are huge. If I understand correctly, these are 10s between checks for IDE status?
2. Names should have `_MS` suffix.

Reply:
1. that is correct - i think that is what it was like on the old frontend
2. will fix asap

re: 1 - it seems like the backend may get overloaded if these are sent too quickly. in particular i tested the CHECK_RESOURCES_INTERVAL and multiple were returning as successful because the old interval was a lot shorter than the response time

Reply:
also worth noting that instances currently still take ~90s to start up so i think the 10 second interval isn't such a big deal, at least for now

Reply:
lets check resources every 1second and status interval every 5 seconds